messages/plugin-missing.txt

```diff
@@ -0,0 +1,9 @@
+ESLint couldn't find the plugin "eslint-plugin-<%- pluginName %>". This can happen for a couple different reasons:
```

Comment:
We normalize all plugin names, right. So plugin name should always include `eslint-plugin-` prefix already. Also, this would work for scoped packages.